quiz.rb

```diff
@@ -14,24 +14,31 @@
 # Take a string and return an array of normalized words.
 # normalized words contain no punctuation and are all lower case
 def normalize_words words
+  paragraph.downcase.delete("." "," " ")
```

Comment:
be careful that the arguments in your declaration and the arguments inside of your function agree. in these instances, you are declaring `words` but manipulating `paragraph`. it works here, but in many instances it wont